CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 Get upgrade notes from Sprockets 3.x to 4.x at https://github.com/rails/sprockets/blob/master/UPGRADING.md
 
-- Fix `Sprockets::Server` to return response headers to compatible with with Rack::Lint 2.0.
```

Comment:
This was a reverting of the lower case headers which caused a performance issue - but this was resolved in rack 2.2.4 according to https://github.com/rails/sprockets/issues/746#issuecomment-1173478546